0001-kaa-protocol/README.md

```diff
@@ -6,40 +6,18 @@ editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
 contributors: Alexey Gamov <agamov@cybervisiontech.com>, Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
 ---
 
-- [Introduction](#introduction)
-- [Language](#language)
-- [Requirements and constraints](#requirements-and-constraints)
-- [Use cases](#use-cases)
-  - [UC1: individually authenticated device](#uc1-individually-authenticated-device)
-  - [UC2: mass production device](#uc2-mass-production-device)
-  - [UC3: actor gateway](#uc3-actor-gateway)
-  - [UC4: forward proxy](#uc4-forward-proxy)
-- [Design](#design)
-  - [Extensions](#extensions)
-  - [Resource path format](#resource-path-format)
-- [Extension design guidelines](#extension-design-guidelines)
-  - [Payload format specifier](#payload-format-specifier)
-  - [Request/response pattern](#requestresponse-pattern)
-  - [Security](#security)
-- [Open questions](#open-questions)
-  - [Topic name aliases](#topic-name-aliases)
-  - [Status topics](#status-topics)
-  - [Endpoint migration](#endpoint-migration)
-    - [UC1](#uc1)
-    - [UC2](#uc2)
-  - [Reporting errors](#reporting-errors)
-  - [Security](#security)
+<!-- toc -->
 
 ## Introduction
 This document describes general requirements, principles, design, and guidelines for **Kaa protocol** (KP) version 1.
-KP is a standard protocol designed to connect client applications and endpoints (colloquially — devices) to a Kaa server.
+KP is a standard protocol designed to connect client applications and endpoints to a Kaa server.
 
 ## Language
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
 The following terms and definitions are used in this RFC.
 
-- **Endpoint**: an entity managed by Kaa platform.
+- **Endpoint**: an entity managed by Kaa platform. <!-- TODO(ashmalko): We need to define this more specifically. There are many entities managed by Kaa, most of which are not endpoints. -->
```

Comment:
I would go for both. Though, I would try to avoid TODOs in RFCs after they stabilize.